docs/herramientas-test.md

```diff
@@ -0,0 +1,102 @@
+# Herramientas para testing
+
+## Metodologías seguidas
+
+Para el desarrollo de tests, se van a seguir dos metodologías:
+
+1. TDD: determina cuándo se escriben los tests (antes de escribir el código de la app)
+2. BDD: determina el tipo de aserciones utilizadas (deben usar una sintaxis más cercana
+al lenguaje natural)
+
+## Criterios de selección
+
+Se han tenido en cuenta los siguientes criterios para la elección de herramientas:
+
+1. *Estándares y conformidad*: que cumpla los estándares relevantes de la industria
+de desarrollo de software, que sea compatible con el lenguaje (TypeScript) y el runtime
+(Node.js) elegidos.
+2. *Comunidad y popularidad*: una comunidad grande y activa; es decir con una gran cantidad
+de contribuidores y un elevado índice de uso entre los usuarios, casi siempre implica
+gran disponibilidad de recursos y ayuda de otros usuarios.
+3. *Mantenimiento*: se considerará que tiene un buen soporte en base a la frecuencia
+de actualizaciones y últimas versiones de la herramienta a evaluar.
+4. *Mejores prácticas*: que las herramientas sigan buenas prácticas de software
+como la capacidad de mantener casos de prueba de forma eficiente, la ejecución
+paralela y la creación de informes concisos.
+
+## Aserciones
+
+Las aserciones o matchers son funciones que comparan la salida obtenida con
+la esperada, incluyendo mensajes si la comparación es positiva.
+
+Teniendo en cuenta que debe ser compatible con el enfoque de BDD, las opciones consideradas
+han sido las siguientes:
+* **Chai**: es una biblioteca de aserciones flexible que se puede utilizar con diversos estilos
+de BDD. Ofrece las interfaces *expect*, *should*, y *assert*, lo que permite elegir el estilo
+que mejor se adapte a las preferencias de uno.
+* **Assert Node.js**: Node.js incluye un módulo de aserciones nativo llamado assert. Este módulo
+proporciona funciones simples para realizar aserciones en entornos Node.js. Aunque es básico en
+comparación con algunas bibliotecas externas.
+* **Jasmine**: es un framework all-in-one que incluye tanto el marco de pruebas como la biblioteca de
+aserciones. Utiliza un estilo de desarrollo conducido BDD que enfatiza la legibilidad y la comprensión
+del código de prueba. Su propio conjunto de matchers para realizar aserciones de manera clara y expresiva.
+* **AVA**: es un marco de pruebas completo, pero también cuenta con sus propias funciones de aserción,
```

Comment:
Por favor, incluid enlaces a las bibliotecas que estéis usando. `ava` tiene aserciones, pero es un marco de pruebas, como tú misma dices. ¿Has considerado la posiblidad de poner cada una en su apartado específico, de forma que sea más sencillo elegir alternativas?

Reply:
Corregido en: https://github.com/marinajcs/asignacionTareas/pull/24/commits/480128a7f2b30f8dea56e29ab5c9a1f8a4e85c95